README.md

```diff
@@ -167,10 +181,26 @@ Facebook has adopted a Code of Conduct that we expect project participants to ad
 
 Read our [contributing guide](CONTRIBUTING.md) to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes to Jest.
 
-### Good First Issues
+### [Good First Issues](https://github.com/facebook/jest/labels/Good%20First%20Issue%20%3Awave%3A)
 
 To help you get your feet wet and get you familiar with our contribution process, we have a list of [good first issues](https://github.com/facebook/jest/labels/Good%20First%20Issue%20%3Awave%3A) that contain bugs which have a relatively limited scope. This is a great place to get started.
 
-### License
+## Credits
+
+This project exists thanks to all the people who [contribute](CONTRIBUTING.md). <a href="graphs/contributors"><img src="https://opencollective.com/jest/contributors.svg?width=890&button=false" /></a>
```

Comment:
The image is not working at the moment. Probably expected, but I wanted to take a note anyway

Reply:
@monkeywithacupcake any ideas? I believe this is all that's left

Reply:
I can't figure it out. That should be circle images of all contributors. 
Lemme ping engineering team

Reply:
Thanks @monkeywithacupcake!